### PR TITLE
Update codeowners

### PR DESCRIPTION
**Pull request by Sherry-Lin**

### CODE REVIEW

**Comment by guowillyy** [comment]
mpeg2d owner should be Will and Yan.

**Comment by Sherry-Lin** [comment]
> mpeg2d owner should be Will and Yan.

Thanks for correction. Updated patch.

File: CODEOWNERS [hunk 1]
```diff
@@ -76,51 +74,49 @@ CODEOWNERS   @Intel-Media-SDK/mediasdk-admin
...
+/_studio/hevc_fei/ @Intel-Media-SDK/mediasdk-admin thomas.li@intel.com apple.wang@intel.com
+/samples/sample_hevc_fei/ @Intel-Media-SDK/mediasdk-admin thomas.li@intel.com apple.wang@intel.com
+/_studio/mfx_lib/encode_hw/h265/src/*fei* @Intel-Media-SDK/mediasdk-admin thomas.li@intel.com apple.wang@intel.com
+/_studio/mfx_lib/encode_hw/h265/include/*fei* @Intel-Media-SDK/mediasdk-admin thomas.li@intel.com apple.wang@intel.com
 
 # enctools:
-/_studio/enctools/ @Intel-Media-SDK/mediasdk-admin timofei.a.kulakov@intel.com olga.golikova@intel.com
-/_studio/enctools/aenc @Intel-Media-SDK/mediasdk-admin sergey.plotnikov@intel.com
-/_studio/shared/include/mfxenctools-int.h @Intel-Media-SDK/mediasdk-admin timofei.a.kulakov@intel.com olga.golikova@intel.com
+/_studio/enctools/ @Intel-Media-SDK/mediasdk-admin dmitry.v.rogozhkin@intel.com
```
Comment (dvrogozh) [inline CODEOWNERS:99]: We need appropriate owner for enctools part. I am not the one. sergey.plotnikov@intel.com might be the candidate.

**Review by dvrogozh — CHANGES_REQUESTED** [review]

**Comment by Sherry-Lin** [comment]
@dvrogozh thanks. Updated patch.

**Review by dvrogozh — APPROVED** [review]
Looks good.

**Pull request merged**